test/powershell/Modules/Microsoft.PowerShell.Security/CmsMessage.Tests.ps1

```diff
@@ -271,16 +271,28 @@ Describe "CmsMessage cmdlets thorough tests" -Tags "Feature" {
         $errors[0].FullyQualifiedErrorId | Should Be "NoCertificateFound"
     }
 
-    It "Verify error when encrypting to wrong cert" {
-        $errors = $null
-        $goodCerts = @(Get-ChildItem Cert:\currentuser\My -DocumentEncryptionCert | ForEach-Object Thumbprint)
-        $badCert = Get-ChildItem Cert:\currentuser\My | ? { $_.Thumbprint -notin $goodCerts } | Select -First 1
+    Context "Verify error when encrypting to wrong cert" {
+        BeforeAll {
+            if ($IsWindows)
+            {
+                $goodCerts = @(Get-ChildItem Cert:\currentuser\My -DocumentEncryptionCert | ForEach-Object Thumbprint)
+                $badCert = Get-ChildItem Cert:\currentuser\My | ? { $_.Thumbprint -notin $goodCerts } | Select -First 1
```

Comment:
Unlike my dev machines, it seems AppVeyor builders don't have any certs in `cert:\currentuser\my`. I probably should generate a bad cert for this test.